lib/endpoints/volumes.js

```diff
@@ -280,6 +291,41 @@ function validateCreateVolumeInput(req, res, next) {
     }
 }
 
+/*
+ * Translate any given `affinity` rules into "locality hints" that
+ * sdc-designation (aka DAPI) currently understands. Eventually it is
+ * planned that DAPI will support affinity rules natively, and we'll only
+ * need to validate the affinity rules here.
+ */
```

Comment:
Didn't DAPI understand affinity rules since https://github.com/joyent/sdc-designation/commit/f2b4e8dac18b400a64fedb27db869b8d32ef009b?

Reply:
Thanks for pointing that out - I had just blindly copied from machines.js.

I will update both these comments to be more relevant.